conans/client/build/cmake_flags.py

```diff
@@ -273,6 +273,7 @@ def get_definitions(self):
 
         compiler = self._ss("compiler")
         compiler_version = self._ss("compiler.version")
+        version = self._ss("version")
```

Comment:
This doesn't look good, ``version`` is not a ``setting``, so this will be always ``None``